### DISCUSSION TITLE
Virtual gallery study on Quire-built Bending Lines in NEMA journal

### DISCUSSION THREAD

**Discussion by garrettdashnelson** [thegetty/quire#117] [Show & Tell]
Hi Quire community,

I thought you all might be interested in reading a report on [evaluating digital exhibitions](https://nemanet.org/nemn/spring-2021/gallery-study-without-gallery/#) that was recently published in _New England Museums Now_. This article was written by two Public History interns at the Leventhal Map & Education Center who had begun a project studying our physical gallery when COVID shifted our exhibition online. They pivoted to learning about how to conduct a gallery study in an online format. While the piece isn't specifically about Quire, it does have some generalizable insights that may be useful to others in thinking about how to evaluate these sorts of productions.

**Comment by Erin-Cecele** [comment]
Thank you for sharing this with the community @garrettdashnelson! I look forward to reading it.
